src/server/users/models.js

```diff
@@ -68,16 +68,19 @@ const UserManager = {
         userValue.facebookId = profile.id;
         return profile.id;
       })
-      .then(FacebookManager.getProfileImage)
+      .then(FacebookModel.isDuplicated)
```

Comment:
so it goes to catch when FacebookModel.isDuplicated does reject(err) @ line 75 in Facebook-manager.js?